Extensions/GG2DLLext/GameMakerExtensionFunctionDescriptions.txt

```diff
@@ -1,14 +1,8 @@
-﻿kind dll-stdcall
```

Comment:
Why did the dll-stdcall lines go?

Reply:
They're redundand since the `kind` switch affects all next lines until another one.